src/boilerplate/orchestration/javascript/raw/toOrchestration.ts

```diff
@@ -386,15 +387,22 @@ export const OrchestrationCodeBoilerPlate: any = (node: any) => {
           states.push(` _${decrementedState}_1_oldCommitment = 0`);
         });
       }
-      node.returnParameters.forEach((param: any) =>
-        rtnparams.push(`, ${param.integer}`),
-      );
+
+      node.returnParameters.forEach( (param:any) => {
```

Comment:
```suggestion
      node.returnParameters?.forEach( (param:any) => {
```
Now we actually fill in return params, might be good to check it exists, though not causing any errors on my end